cime_config/user_nl_blom_outvol_reduced

```diff
@@ -0,0 +1,534 @@
+  LYR_ANO3          = 0, 0
+  LYR_DENIT_N2O     = 0, 0
+  LYR_DENIT_NO2     = 0, 0
+  LYR_DENIT_NO3     = 0, 0
+  LYR_DNRA_NO2      = 0, 0
+  LYR_DYNVIS        = 0, 0
+  LYR_NITR_N2O_PROD = 0, 0
+  LYR_NITR_NH4      = 0, 0
+  LYR_NITR_NH4_OM   = 0, 0
+  LYR_NITR_NO2      = 0, 0
+  LYR_NITR_NO2_OM   = 0, 0
+  LYR_PHOSY_NH4     = 0, 0
+  LYR_PHOSY_NO3     = 0, 0
+  LYR_REMIN_AEROB   = 0, 0
+  LYR_REMIN_SULF    = 0, 0
+  LYR_PHYTO         = 0, 0
+  LYR_GRAZER        = 0, 0
+  LYR_DOC           = 0, 0
+  LYR_PHOSY         = 0, 0
+  LYR_PHOSPH        = 0, 0
+  LYR_OXYGEN        = 0, 0
+  LYR_IRON          = 0, 0
+  LYR_ALKALI        = 0, 0
+  LYR_SILICA        = 0, 0
+  LYR_DIC           = 0, 0
+  LYR_POC           = 0, 0
+  LYR_CALC          = 0, 0
+  LYR_OPAL          = 0, 0
+  LYR_CO3           = 0, 0
+  LYR_N2O           = 0, 0
+  LYR_PH            = 0, 0
+  LYR_OMEGAC        = 0, 0
+  LYR_OMEGAA        = 0, 0
+  LYR_PREFO2        = 0, 0
+  LYR_O2SAT         = 0, 0
+  LYR_PREFPO4       = 0, 0
+  LYR_PREFALK       = 0, 0
+  LYR_PREFDIC       = 0, 0
+  LYR_PREFSILICA    = 0, 0
+  LYR_DICSAT        = 0, 0
+  LYR_NATDIC        = 0, 0
+  LYR_NATALKALI     = 0, 0
+  LYR_NATCO3        = 0, 0
+  LYR_NATCALC       = 0, 0
+  LYR_NATPH         = 0, 0
+  LYR_NATOMEGAC     = 0, 0
+  LYR_NATOMEGAA     = 0, 0
+  LYR_DIC13         = 0, 0
+  LYR_DIC14         = 0, 0
+  LYR_D13C          = 0, 0
+  LYR_D14C          = 0, 0
+  LYR_BIGD14C       = 0, 0
+  LYR_POC13         = 0, 0
+  LYR_DOC13         = 0, 0
+  LYR_CALC13        = 0, 0
+  LYR_PHYTO13       = 0, 0
+  LYR_GRAZER13      = 0, 0
+  LYR_CFC11         = 0, 0
+  LYR_CFC12         = 0, 0
+  LYR_SF6           = 0, 0
+  LYR_NOS           = 0, 0
+  LYR_WPHY          = 0, 0
+  LYR_WNOS          = 0, 0
+  LYR_EPS           = 0, 0
+  LYR_ASIZE         = 0, 0
+  LYR_BROMO         = 0, 0
+  LYR_SHELFAGE      = 0, 0
+  LYR_DP@diabgc     = 0, 0
+
+! Save level-fields only annually
+
+  LVL_AGG_AVDC      = 0, 0
+  LVL_AGG_AVDP      = 0, 0
+  LVL_AGG_AVRHOP    = 0, 0
+  LVL_AGG_B         = 0, 0
+  LVL_AGG_DF        = 0, 0
+  LVL_AGG_DMAX      = 0, 0
+  LVL_AGG_STICK     = 0, 0
+  LVL_AGG_STICKF    = 0, 0
+  LVL_AGG_VPOR      = 0, 0
+  LVL_AGG_VRHOF     = 0, 0
+  LVL_AGG_WS        = 0, 0
+  LVL_PHYTO         = 0, 2
+  LVL_GRAZER        = 0, 2
+  LVL_DOC           = 0, 2
+  LVL_PHOSY         = 0, 2
+  LVL_PHOSY_NH4     = 0, 0
+  LVL_PHOSY_NO3     = 0, 0
+  LVL_PHOSPH        = 0, 2
+  LVL_OXYGEN        = 0, 4
+  LVL_IRON          = 0, 2
+  LVL_ALKALI        = 0, 2
+  LVL_ANH4          = 0, 2
+  LVL_ANMX_N2_PROD  = 0, 0
+  LVL_ANMX_OM_PROD  = 0, 0
+  LVL_ANO2          = 0, 2
+  LVL_ANO3          = 0, 2
+  LVL_SILICA        = 0, 2
+  LVL_DIC           = 0, 2
+  LVL_POC           = 0, 2
+  LVL_CALC          = 0, 2
+  LVL_OPAL          = 0, 2
+  LVL_CO3           = 0, 2
+  LVL_N2O           = 0, 2
+  LVL_PH            = 0, 2
+  LVL_OMEGAC        = 0, 2
+  LVL_OMEGAA        = 0, 2
+  LVL_PREFO2        = 0, 4
```

Comment:
any reasons to have this set to 4? (similar oxygen above).

Reply:
Thanks for the info! (I guessed so, but wasn't sure - so thanks for confirmation).